debian/changelog

```diff
@@ -1,5 +1,8 @@
 cloud-init (22.1-14-g2e17a0d6-0ubuntu1~21.10.1) impish; urgency=medium
 
+  * d/control:
```

Comment:
This needs to be in a separate unreleased changelog entry since 22.1-14-g2e17a0d6-0ubuntu1~21.10.1 is already published.  `dch -v 22.1-14-g2e17a0d6-0ubuntu1~21.10.4` and move the d/control content up into the new UNRELEASED entry. Probably generally a good practice to keep changelog commits separate from functional changes.